README.md

```diff
@@ -164,6 +176,104 @@ ember exam --split=3 --partition=3
+
+```bash
+$ ember exam --split=<num> --partition=<num> --parallel=<num> --load-balance
+```
+
+This command will split test files and load-balance tests from the specified partition across the browsers. For example --split=2 -partition=1 --parallel=3 --load-balance, the complete list of test files are split into two halves. With the first half of the list load balanced against three browsers. The output will look something like below:
+
+```bash
+# ember exam --split=2 --partition=1 --parallel=3 --load-balance
+ok 1 Chrome 66.0 - Exam Partition 1 - browser Id 1 - some test
+ok 2 Chrome 66.0 - Exam Partition 1 - browser Id 2 - another test
+ok 3 Chrome 66.0 - Exam Partition 1 - browser Id 3 - some the other test
+```
+
+
+`load-balance` is currently only supported in CI mode and due to the reason `no-launch` can not be used with `load-balance`.
+
+**Note 1** _You must be using ember-cli version 3.2.0 or greater for load-balnce and test failure reproduction features to work properly._
+**Note 2** _You must be using ember-qunit version 4.1.1 or greater for this feature to work properly._
+**Note 3** _You must be using qunit version 2.8.0 or greater for this feature to work properly._
+**Note 4** _This feature is not currently supported by Mocha._
+
+##### Test Failure Reproduction
+
+Due to the dynamic nature of the `load-balance` option, test file execution order could vary between runs. To reproduce a past test execution the execution must be recorded and passing `--write-execution-file`, in short `--wef` allows generating a JSON file which enables to rerun the past test execution. The option is only allowed when `load-balance` is passed.
+
+```bash
+# The command will load test balanced with <num> of browser(s). After the test suite execution It will generate a test-execution json file.
+$ ember exam --parallel=<num> --load-balance --wef
+$ ember exam --parallel=<num> --load-balance --write-execution-file
+```
+
+The file is stored in the root directory and the naming structure is `test-execution-<timestamp>.json`.
+To replay the test execution for particular browser(s), do the following:
+
+```bash
+# The command will read a test execution file specified for `replay-execution` and execute a browser Id(s) from `replay-browser`
+$ ember exam --replay-execution=[string] --replay-browser=[num]
+```
+
+`replay-execution` allows you to specify a path to the json file to run execution against and `replay-browser` is to specify browser ID(s) to execute.
+
+```bash
+# The command will read test-execution-000000.json and load the list of module mapped to browserId 1
+$ ember exam --replay-execution=test-execution-000000.json --replay-browser=1
+```
+
+The above command will read `test-execution-000000.json` and load the list of modules which is mapped by browser ID #1.
+
+`replay-browser` can be an array of broweser IDs. For instance `--replay-broweser=1,2` will bring two browsers and execute a list of modules which ran by previous browser #1 and #2.
```

Comment:
> `replay-browser` can be an array of broweser IDs. For instance `--replay-broweser=1,2` will bring two browsers and execute a list of modules which ran by previous browser #1 and #2.

I'd change this to:

`replay-browser` can be an array of browser IDs. For instance `--replay-broweser=1,2` will start two browsers and execute a list of modules which were previously run by browsers #1 and #2.